packages/core/phase1/schemas/pncQueryResult.ts

```diff
@@ -8,12 +8,12 @@ export const pncDisposalSchema = z.object({
   qtyUnitsFined: z.string().optional(),
   qualifiers: z.string().optional(),
   text: z.string().optional(),
-  type: z.number()
+  type: z.number().optional()
 })
 
 export const pncAdjudicationSchema = z.object({
   verdict: z.string(),
-  sentenceDate: z.coerce.date(),
+  sentenceDate: z.coerce.date().optional(),
```

Comment:
Phase 1 comparison tests passed after making `sentenceDate` and `type` optional in the schema and updating Phase 1 code.